src/main/groovy/com/avast/gradle/dockercompose/ComposeExecutor.groovy

```diff
@@ -100,7 +100,11 @@ class ComposeExecutor {
     }
 
     Iterable<String> getContainerIds(String serviceName) {
-        execute('ps', '-q', serviceName).readLines()
+        try {
+            execute('ps', '-q', serviceName).readLines()
+        } catch (RuntimeException ignored) {
+            return []
```

Comment:
Could we please be more specific here, and rethrow if an unexpected exception is thrown? 🙏

Reply:
I think I've found a better way to guard against that, so we can skip the exception handling altogether.  Thanks for prodding me into reinvestigating that.  👍

Reply:
Yeah, I think that there could be a possibility to _ignore_ the exit-code of the executed program.